bpf/lib/encap.h

```diff
@@ -219,15 +214,6 @@ encap_and_redirect_netdev(struct __ctx_buff *ctx, struct tunnel_key *k,
 	if (!tunnel)
 		return DROP_NO_TUNNEL_ENDPOINT;
 
-#ifdef ENABLE_IPSEC
-	if (tunnel->key) {
-		__u8 key = get_min_encrypt_key(tunnel->key);
-
-		return encap_and_redirect_nomark_ipsec(ctx, key,
-						       tunnel->node_id,
-						       seclabel);
-	}
-#endif
```

Comment:
Ack.

Note that as we removed both callers in this patch, this should allow us to also delete `encap_and_redirect_nomark_ipsec()`. And `set_encrypt_key_meta()` as well I believe.